sway/sway-ipc.7.scd

```diff
@@ -1702,6 +1705,58 @@ event is a single object with the following properties:
 }
 ```
 
+## 0x80000015. INPUT
+
+Sent when something related to the input devices changes. The event is a single
+object with the following properties:
+
+[- *PROPERTY*
+:- *DATA TYPE*
+:- *DESCRIPTION*
+|- change
+:  string
+:[ What has changed
+|- input
+:  object
+:  An object representing the input that is identical the ones GET_INPUTS gives
+
+The following change types are currently available:
+[- *TYPE*
+:- *DESCRIPTION*
+|- added
+:[ The input device became available
+|- removed
+:  The input device is no longer available
+|- xkb_keymap_changed
+:  (Keyboards only) The keymap for the keyboard has changed
+|- xkb_layout_changed
+:  (Keyboards only) The effective layout in the keymap has changed
+|- libinput_config_changed
+:  (libinput device only) A libinput config option for the device changed
+
+*Example Event:*
+```
+{
+	"change": "xkb_layout_changed",
```

Comment:
It seems like i doesn't add a `_changed` suffix when properties change (see the `window` event for instance). For consistency this should probably just be the name of the property.

Reply:
Changed